index.html

```diff
@@ -3,9 +3,24 @@
     <head>
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link
+            rel="stylesheet"
+            href="C:\Users\Carte\github-sample\community-html-css-page-1\Styles sheet Community HTML CSS.css" />
```

Comment:
```suggestion
            href="style.css" />
```
you can use a relative path here to the style.css instead of an absolute as the stylesheet will always reside in the project folder.

Reply:
You just answered my pervious question above.  Separate directories for each project.  

Question:  If the style sheet resides in the same directory as the markup, the relative path works?

Reply:
Hmm, what do you mean by markup?

Reply:
Using a relative path in an index.html file, means where is that stylesheet relative to that file. Same folder would just be ```style.css```. If in a separate folder you could do ```/styles/style.css```. If for some reason in a folder above you could do ```../style.css```

Reply:
Markup meaning html.   Colt Steele calls html markup.  That's where I picked it up.  I'm trying to sound in the know... :)

I committed and pushed the markup (html) in VSCode- looked successful.   A little confused. I'll need to do another pull request so you can see my  current updates  approx. 15min ago? 

Also- Since I wanted to complete the html, I didn't merge or commit your pull requests.  (the pull request process is a little confusing to me). 

Question: Every time I commit and push a change in VSCode, I'll need to do a pull request in github?

Reply:
Neverminded last Questions.  I see my commits got pushed right here.   I'm getting GIT and GitHub much better now that I did this project.  This is VERY helpful Travis.  Thank you.